.github/workflows/test-tgc.yml

```diff
@@ -1,9 +1,6 @@
 name: TGC Build and Unit Test
 
-permissions:
-  actions: read
-  contents: read
-  statuses: write
+permissions: read-all
```

Comment:
If any permission is specified the remaining are set to none
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Just context on why it was written in this way.

Reply:
Acknowledged - thanks! I think I'd rather switch to read-all for consistency with go/github-docs/security-recommendations and other workflows. Although this pattern is not explicitly discouraged - I don't think it adds any benefit since this is a public repo.